client/packages/host/src/routers/ManageRouter.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { RouteBuilder, Navigate, useMatch } from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+
+const NameService = React.lazy(
+  () => import('@openmsupply-client/system/src/Name/Service')
+);
+
+const fullFacilitiesPath = RouteBuilder.create(AppRoute.Manage)
+  .addPart(AppRoute.Facilities)
+  .addWildCard()
+  .build();
+
+export const ManageRouter: FC = () => {
+  const gotoCustomers = useMatch(fullFacilitiesPath);
+
+  if (gotoCustomers) {
+    return <NameService />;
+  }
```

Comment:
:nit
```suggestion
export const ManageRouter: FC = () => {
  const gotoFacilities = useMatch(fullFacilitiesPath);

  if (gotoFacilities) {
    return <NameService />;
  }
```

should that be `gotoNames` or `<FacilityService />` 🤔